sui/docs/channel.md

```diff
@@ -0,0 +1,247 @@
+    destination_token_address: vector<u8>,
+    token_manager_type: TokenManagerType,
+    link_params: vector<u8>,
+): MessageTicket
+```
+
+Functionality:
+1. Re-derives the token ID using the same `deployer` Channel and `salt` (validates the deployer)
+2. Verifies the token exists and is registered as a custom token
+3. Constructs a LINK_TOKEN message to be sent via ITS Hub
+4. Uses the ITS contract's internal `Channel` to send the constructed message
+
+**Security Considerations**:
+
+- **Salt Uniqueness**: The salt must be unique per token linking operation to prevent collisions
+- **Channel Ownership**: Only the holder of the deployer `Channel` can link tokens registered with that `Channel`
+- **Immutability**: Once a token ID is claimed by a deployer/salt pair, it cannot be re-registered
+- **Salt Format**: On Sui, the `salt` must be exactly 32 bytes (matching Sui address format)
+
+### Example Token Linking Flow
+
+```move
+// User creates their deployer channel
+let deployer_channel = channel::new(ctx);
+
+// Define a unique salt (0x + 64 chars.)
+let salt = bytes32::new(0x0000...0001);
+
+// Register the custom token
+let (token_id, treasury_cap_reclaimer) = its.register_custom_coin(
+    &deployer_channel,  // Channel used for token ID derivation
+    salt,
+    &coin_metadata,
+    coin_management,
+    ctx
+);
+
+// Link to destination chain
+let message_ticket = its.link_coin(
+    &deployer_channel,  // Same channel ensures same token_id
+    salt,               // Same salt ensures same token_id  
+    destination_chain,
+    destination_token_address,
+    token_manager_type,
+    link_params
+);
+```
+
+### Receiving Unlinked Tokens
+
+To receive a `LinkToken` message from another chain, the Sui ITS contract (the destination) must prepare the "unlinked coin" before the link message arrives. This is done by giving the unlinked coin to ITS. 
+
+If the coin's `TreasuryCap` is given to ITS in the `give_unlinked_coin` transaction, ITS creates a new `Channel` that will validate the deployer for coin linking; and, in the case of MINT_BURN token managers (`CoinManagement`), ITS makes this `Channel` a distributor of the coin.
+
+```move
+public fun give_unlinked_coin<T>(
+    self: &mut InterchainTokenService,
+    token_id: TokenId,  // The expected token ID from the link operation
+    coin_metadata: &CoinMetadata<T>,
+    treasury_cap: Option<TreasuryCap<T>>,
+    ctx: &mut TxContext,
+): (Option<TreasuryCapReclaimer<T>>, Option<Channel>)
+```
+
+This is particularly important for MINT_BURN type token managers, where the workflow for `give_unlinked_coin` is:
+
+1. ITS creates a new `Channel`
+2. ITS adds the newly created `Channel` as distributor for the coin
+3. `give_unlinked_coin` returns the `Channel` to the caller, giving them distributor privileges. This ensures that the deployer on the destination chain receives appropriate control over the linked token once the linking is complete.
+
+## Creating & Using Channels
```

Comment:
How else could I create a channel other than via `interchain-transfer` or `register-custom-coin`? 

If I am for example linking a custom coin on sui to another custom token on ethereum (with ethereum being the home chain) i wouldn't technically be using the `register-custom-coin` fn so how would the channel be created in that scenario?